src/systems/systems/src/hydraulic/linear_actuator.rs

```diff
@@ -628,7 +797,7 @@ impl LinearActuatedRigidBodyOnHingeAxis {
             control_arm_actual: control_arm,
             actuator_extension_gives_positive_angle: false,
             anchor_point,
-            position: min_angle,
+            position: init_angle,
             speed: AngularVelocity::new::<radian_per_second>(0.),
             acceleration: AngularAcceleration::new::<radian_per_second_squared>(0.),
```

Comment:
While it is pretty clear to me now, it took me some time to realize that position is actually an angle. I think we could improve clarity a bit by renaming it everywhere.

I can see how `position` might be more concrete than `angle`, but maybe with `current_angle`? It would then also make sense to rename `speed` and `acceleration` to their respective rotational counterparts.

Reply:
Yeah I'm only rotating things so to me it's clear :D

Clearer using angular_xxx naming I hope now!